clients/cloud/python/producer_ccsr.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+#
+# Copyright 2019 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# =============================================================================
+#
+# Produce messages to Confluent Cloud
+# Using Confluent Python Client for Apache Kafka
+# Writes Avro data, integration with Confluent Cloud Schema Registry
+#
+# =============================================================================
+
+from confluent_kafka import Producer, KafkaError
+from confluent_kafka.admin import AdminClient, NewTopic
+from confluent_kafka.avro import AvroProducer
+import json
+import ccloud_lib
+
+
+if __name__ == '__main__':
+
+    # Initialization
+    args = ccloud_lib.parse_args()
+    config_file = args.config_file
+    topic = args.topic
+    conf = ccloud_lib.read_ccloud_config(config_file)
+
+    # Create AvroProducer instance
+    p = AvroProducer({
+           'bootstrap.servers': conf['bootstrap.servers'],
+           'sasl.mechanisms': 'PLAIN',
+           'security.protocol': 'SASL_SSL',
+           'sasl.username': conf['sasl.username'],
+           'sasl.password': conf['sasl.password'],
+           'schema.registry.url': conf['schema.registry.url'],
+           'schema.registry.basic.auth.credentials.source': conf['basic.auth.credentials.source'],
+           'schema.registry.basic.auth.user.info': conf['schema.registry.basic.auth.user.info']
+    }, default_key_schema=ccloud_lib.schema_key, default_value_schema=ccloud_lib.schema_value)
+
+    # Create topic if needed
+    # Examples of additional admin API functionality:
+    # https://github.com/confluentinc/confluent-kafka-python/blob/master/examples/adminapi.py
+    a = AdminClient({
```

Comment:
If we put Admin client in ccloud_lib we could reuse it in future examples.

Reply:
Excellent suggestion.  I actually already have a need now -- two different producers use the AdminClient code.